scripts/auto-config_UWARL_catkin_ws.zsh

```diff
@@ -95,6 +95,12 @@ case $UWARL_ROBOT_PC_NAME in
         install_libbarrett_if_not
         install_librealsense_if_not # for Intel Sensors
     ;;
+    "corey-ThinkPad")
+        load_submodules "${SUBMODULES_FOR_Koda_Laptop[@]}"
+        install_misc_utilities # misc apt 
+        install_libbarrett_if_not
```

Comment:
you don't need libbarrrett, it is for WAM, please copy from SUMMIT
```
"ADLINK_MXE211_SUMMIT" )
        ic " - Adlink MXE211 Summit PC detected!" 
        ic " > Loading SUMMIT workspace submodules:"
        load_submodules "${SUBMODULES_FOR_SUMMIT[@]}"
        # install drivers :
        install_pcan_if_not
    ;;
    ```

Reply:
Yes, this can be ignored. If @CazesC wants to make new WAM scripts that can be run using `Libbarrett` library, just make a new branch off [uwarl-libbarrett](https://github.com/UW-Advanced-Robotics-Lab/uwarl-libbarrett).